### ISSUE TITLE
MS-COCO Multilable training data path issue

### ISSUE THREAD

**Issue by aliman80** [Alibaba-MIIL/ML_Decoder#23]
HI thanks for excellent work, 
1.After validating the results i got mAP of 86 , any idea what i am doing wrong here, how to increase the number of iterations .
2. While i was trying to reproduce the trainnig logs i cam across this line in code
data_path_train=data_path_val in train.py  at line no 56 , so this makes train and val similar then are we basically training the results on same validation set?

Thanking to you in anticipation

**Comment by aliman80** [comment]
Thanks very much.Excellent
